.github/workflows/docs.yml

```
@@ -0,0 +1,17 @@
+name: Docs build
+
+on: [push, pull_request]
+
+jobs:
+  docs_build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+      # TODO: Dependencies come from environment.yml.
+      # How should they be shared here? Install everything?
+      - run: pip install sphinx==4.2.0 sphinx-rtd-theme==1.0.0 nbclean==0.3.2 nbsphinx==0.8.8
```

```suggestion
      - run: |
        pip install sphinx==4.2.0 sphinx-rtd-theme==1.0.0 nbclean==0.3.2 nbsphinx==0.8.8
        pip install -e .
```

The vitessce package defined in this repo needs to be installed into the environment before the docs are generated